haya-hayaodeh/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -10,3 +10,11 @@
   conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
 */
 
+for (let i = 99; i>0 ; i--) {
+  
+ console.log(i + " bottles of beer on the wall, " + i + " bottles of beer.");
```

Comment:
Different developers standardize on different numbers of spaces, but 2 spaces is pretty common. A single space can be pretty hard to distinguish, so I recommend increasing that in the future.